another-import-service/import-file-parser.js

```diff
@@ -0,0 +1,55 @@
+import AWS from 'aws-sdk';
+import csv from 'csv-parser';
+
+import config from './config.js';
+
+const s3 = new AWS.S3({ region: config.aws.region }),
+    collectDataForLogging = objKey => {
+        return new Promise((res, rej) => {
+            const results = [];
+
+            s3.getObject({
+                Bucket: config.aws.s3.bucket,
+                Key: objKey
+            }).createReadStream()
+                .on('error', e => {
+                    console.log(e);
+                    rej(e);
+                })
+                .pipe(csv())
+                .on('error', e => {
+                    console.log(e);
+                    rej(e);
+                })
```

Comment:
Subscribe on error only once here is enough. It's not necessary to subscribe two times.